test/integration/install_scripts_int_test.go

```diff
@@ -185,8 +186,10 @@ func (suite *InstallScriptsIntegrationTestSuite) TestInstall_VersionDoesNotExist
 	} else {
 		cp = ts.SpawnCmdWithOpts("powershell.exe", e2e.OptArgs(args...), e2e.OptAppendEnv("SHELL="))
 	}
-	cp.Expect("Could not download")
-	cp.Expect("does-not-exist")
```

Comment:
This line removal was intentional, because not all download failures are due to version errors (invalid branch could be specified for example).